main.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+require_relative 'app'
+
+def main
+  puts "Welcome to School Library App!\nPlease choose an option by enterin a number from 1 to 7:
+    \n1 - List all books\n2 - List all people\n3 - Create a person\n4 - Create a book
+    \n5 - Create a rental\n6 - List all rental for a given id\n7 - Exit"
+  action = gets.chomp.to_i
+
+  if action < 7 && action.positive?
+    start(action)
+  elsif action == 7
+    puts 'Good Bye!'
+  else
+    main
+  end
+end
+
```

Comment:
- [OPTIONAL] The `main.rb` file is the entry point that is invoked at the end of the file. You should Present the user with a list of options to perform here and not in the app.js file.
- This method should do and look similar to the following:

```ruby
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    ...
    puts "7 - Exit"
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
       ...
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()
```